content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
+Create a `webpack.config.js` file in the project root directory, and paste in the following configuration:
+
+```javascript file=webpack.config.js
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path')
+const nodeExternals = require('webpack-node-externals')
+const slsw = require('serverless-webpack')
+const { isLocal } = slsw.lib.webpack
+
+module.exports = {
+  target: 'node',
+  stats: 'normal',
+  entry: slsw.lib.entries,
+  externals: [nodeExternals()],
+  mode: isLocal ? 'development' : 'production',
+  optimization: { concatenateModules: false },
+  resolve: { extensions: ['.js'] },
+  output: {
+    libraryTarget: 'commonjs',
+    filename: '[name].js',
+    path: path.resolve(__dirname, '.webpack'),
+  },
+}
 ```
 
-### Result
-
-If the configuration is successful, you see the following result in the terminal.
-
-```terminal
-✔ Profile "default" has been configured
+If you use TypeScript, insert the following configuration inside `exports`:
+
+```javascript file=webpack.config.js
+// ...initial config
+module.exports = {
+  // ...initial config
+  resolve: { extensions: ['.js', '.ts'] },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader',
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  // ...initial config
+}
 ```
 
-## 5. Deploy your project
-
-You have finished the configuration, so you can now deploy your project with `serverless deploy`.
+Refer to the [Serverless Webpack documentation](https://www.serverless.com/plugins/serverless-webpack) for additional configuration.
 
-### Steps
+</details>
 
-Use the following command:
+To install the dev dependency, use the following command:
 
-```terminal
-serverless deploy
+```sh
+npm install -D serverless-webpack-prisma serverless
 ```
 
-### Result
+In `serverless.yml`, add `serverless-webpack-prisma` to the plugins list, as follows:
 
-Serverless creates the AWS resources and uploads your code, then outputs the service information. For example:
+```yaml file=serverless.yml
+plugins:
+  - serverless-webpack
+  - serverless-webpack-prisma
 
-```code no-copy
-Service Information
-service: prisma-aws-lambda-example
-stage: dev
-region: us-east-1
-stack: prisma-aws-lambda-example-dev
-resources: 39
-api keys:
-  None
-endpoints:
-  GET - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/
-  GET - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/seed
-  GET - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/users
-  POST - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/users
-  GET - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/posts
-functions:
-  status: prisma-aws-lambda-example-dev-status
-  seed: prisma-aws-lambda-example-dev-seed
-  getUsers: prisma-aws-lambda-example-dev-getUsers
-  createUser: prisma-aws-lambda-example-dev-createUser
-  getPosts: prisma-aws-lambda-example-dev-getPosts
-layers:
-  None
```

Comment:
This looks like useful information to include in the new guide to visualize the success of a `serverless deploy`.